### DISCUSSION TITLE
too many wrong password attempts

### DISCUSSION THREAD

**Discussion by WayneLee7** [rustdesk/rustdesk#6492] [General]
Mayday! I changed the password on remote client but forget to remember it before close connection. What should I do now... I've tried some password in my mind until it reports "too many wrong password attempts"...
both of the clients are mine, as well as the relay server...
maybe temporarily set a higher attempt threshold at the relay server? but how.... SOS

**Comment by rustdesk** [comment]
Restart service (--server process) on controlled side or reboot.
